plugins/statistics.koplugin/main.lua

```diff
@@ -1596,6 +1653,15 @@ function ReaderStatistics:getCurrentBookStats()
     return current_duration, current_pages
 end
 
+function ReaderStatistics:getSequenceNumberForPageLabel(label, page_map)
+    for i = 1, #page_map do
+        if page_map[i].label == label then
+            return i
+        end
```

Comment:
May be this is what that should be used to compute curr_page and new_pagecount to get numbers - that I mentionned above.
But it's not cheap... No thought about a better solution at the moment.
This would have been my No1 argument for not implementing this feature... Page labels are not page numbers.